flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecTemporalSort.java

```diff
@@ -119,9 +123,11 @@ protected Transformation<RowData> translateToPlanInternal(
         RowType inputType = (RowType) inputEdge.getOutputType();
         LogicalType timeType = inputType.getTypeAt(sortSpec.getFieldSpec(0).getFieldIndex());
         if (isRowtimeAttribute(timeType)) {
+            System.out.println("createSortRowTime");
```

Comment:
we should log - not write to stdout

Reply:
thanks @Au-Miner - what did you think about the logging idea, where you had the stdouts. Mostly debug maybe some infos? As they were presumably helpful for you when developing this change?